archive/terraform/iam.tf

```diff
@@ -0,0 +1,125 @@
+data "aws_iam_policy_document" "archive_dlcs_get" {
+  statement {
+    effect = "Allow"
+
+    principals {
+      type = "AWS"
+
+      identifiers = [
+        "arn:aws:iam::653428163053:user/echo-fs",
+        "arn:aws:iam::653428163053:user/api",
+      ]
```

Comment:
Can these ids be in variables in the private vars?

Reply:
Bit late now! Seriously I don't think these are dangerous to have in the wild?